plugins/labhub.py

```diff
@@ -409,17 +409,19 @@ def eligible(user, iss):
             if not iss.assignees:
                 if eligible(user, iss):
                     iss.assign(user)
-                    yield ('Congratulations! You\'ve been assigned to the '
-                           'issue. :tada:')
+                    yield ('@{}, congratulations! You\'ve been assigned to the '
+                           'issue. :tada:'.format(user))
                 else:
-                    yield 'You are not eligible to be assigned to this issue.'
+                    yield ('@{}, You are not eligible to be assigned to this'
```

Comment:
Missing space after `this` causes the test to fail.

```suggestion
                    yield ('@{}, You are not eligible to be assigned to this '
```

Reply:
Yeah @abhishalya, small mistakes :P